proxy/pkg/zdmproxy/cqlconn.go

```diff
@@ -440,7 +442,7 @@ func (c *cqlConn) Execute(msg message.Message, ctx context.Context) (message.Mes
 
 func (c *cqlConn) SendHeartbeat(ctx context.Context) error {
 	optionsMsg := &message.Options{}
-	heartBeatFrame := frame.NewFrame(ccProtocolVersion, -1, optionsMsg)
+	heartBeatFrame := frame.NewFrame(c.protocolVersion, -1, optionsMsg)
```

Comment:
access to `c.protocolVersion` should probably be atomic, maybe using `atomic.Value`